docs/src/auth.md

```diff
@@ -71,6 +71,18 @@ page.click('text=Submit')
 # Verify app is logged in
 ```
 
+```csharp
+var page = await browser.NewPageAsync();
+await page.GoToAsync("https://github.com/login");
```

Comment:
GoToAsync is now Goto

Reply:
Yeah, I was going to go and retrospectively fix them all in one swoop down the line. The fact it's still here is a result of me checking/compiling the samples as I write them, and I have an _old_-ish version of the API.